### ISSUE TITLE
Option to turn off the import mechanism

### ISSUE THREAD

**Issue by felixhayashi** [Jermolene/TiddlyWiki5#1482]
When tiddlywiki is used as a read-only website, it is not favourable to have the import mechanism activated since the user may accidentally activate it by dragging a link. An option to turn the import mechanism off would be great.

**Comment by Jermolene** [comment]
Thanks @felixhayashi I've actually updated #1213 to explicitly state that the proposed read-only mode should disable drag-and-drop import.

**Comment by felixhayashi** [comment]
ah sry should have updated it there.

**Comment by Jermolene** [comment]
Not at all @felixhayashi it was my suggestion to open a new ticket, it was only once I'd read it that I realised it belonged in #1213. Sorry for the makework.

**Comment by felixhayashi** [comment]
Ok. No problem :)

**Comment by danielo515** [comment]
So this is not implemented already, right?

**Comment by Jermolene** [comment]
No, it's not implemented yet I'm afraid @danielo515
